docs/user/bots.rst

```diff
@@ -583,6 +583,21 @@ Rsync
 * `temp_directory`: The temporary directory for rsync, by default `$VAR_STATE_PATH/rsync_collector`. `$VAR_STATE_PATH` is `/var/run/intelmq/` or `/opt/intelmq/var/run/`.
 * `rsync_path`: The path of the file to process
 
+Shadowserver API
+^^^^^^^^^^^^^^^^
+
+**Information**
+
+* `name`: intelmq.bots.parsers.shadowserverapi.collector
+* `description`: Connects to the `Shadowserver API <https://www.shadowserver.org/what-we-do/network-reporting/api-documentation/>`_, requests a list of all the reports for a specific country and processes the ones that are new.
+
+**Configuration Parameters**
+
+* `country`: The country you want to downloads the reports for
+* `apikey`: Your Shadowserver API key
+* `secret`: Your Shadowserver API secret
+* `types`: A list of types of reports you want to process. If you leave this empty, all the available reports will be downloaded and processed.
```

Comment:
Please add that this can either be a list or a string with comma-separated values. Also some hint which values are appropriate would be helpful.

Reply:
Please add that the cache parameters apply as well (and for what) and the default TTL is 10 days.

Reply:
done